Instagram/Instagram/PostTableViewCell.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PostTableViewCell.swift
+//  Instagram
+//
+//  Created by angelina on 17.10.2022.
+//
+
+import UIKit
+
+///  Класс ячейки с постом
+final class PostTableViewCell: UITableViewCell {
+
+    // MARK: - Outlet
+    @IBOutlet weak var avatarImageView: UIImageView!
```

Comment:
```suggestion
    @IBOutlet private weak var avatarImageView: UIImageView!
```
UI элементы должны быть приватными